node/src/gql/types/lesson-info.ts

```diff
@@ -0,0 +1,25 @@
+/*
+This software is Copyright ©️ 2020 The University of Southern California. All Rights Reserved. 
+Permission to use, copy, modify, and distribute this software and its documentation for educational, research and non-profit purposes, without fee, and without a written agreement is hereby granted, provided that the above copyright notice and subject to the full license file found in the root of this software deliverable. Permission to make commercial use of this software may be obtained by contacting:  USC Stevens Center for Innovation University of Southern California 1150 S. Olive Street, Suite 2300, Los Angeles, CA 90115, USA Email: accounting@stevens.usc.edu
+
+The full terms of this copyright and license should always be found in the root directory of this software deliverable as "license.txt" and if these terms are not found with this software, please contact the USC Stevens Center for the full license.
+*/
+import {
+  GraphQLString,
+  GraphQLObjectType,
+  GraphQLID,
+  GraphQLBoolean,
+} from 'graphql';
+
+export const LessonInfoType = new GraphQLObjectType({
+  name: 'LessonInfo',
+  fields: {
+    deleted: { type: GraphQLBoolean },
+    id: { type: GraphQLID },
```

Comment:
I think this should be _id?

Reply:
05e2670 mongoose adds a virtual `id` property for all documents, and I think we should always prefer the plain `id` over `_id`, at least for the external interface.

I added a test to show that it does return the id for a lessonInfo query. I also removed `deleted` as a field, since no reason to have it